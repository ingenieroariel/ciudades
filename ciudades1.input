United Kingdom
Paris
Miami
Barranquilla
