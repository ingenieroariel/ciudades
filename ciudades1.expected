U.K. of Great Britain and Northern Ireland
Miami-DADE, Florida, United States of America
Paris, Ile-de-France, France
Barranquilla (dist.port., Atlantico, Colombia
